src/wget_agent/agent/libwget_agent.a.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: © Fossology contributors
+
+SPDX-License-Identifier: GPL-2.0-only
```

Comment:
The file `src/wget_agent/agent/libwget_agent.a` is generated by the `src/wget_agent/agent/Makefile` and should ideally be added to the `.gitignore`

Reply:
See: https://github.com/fossology/fossology/blob/20ca3a482ce18e85859e92842e94fe06bb226237/.gitignore#L7

Reply:
Corrected.